### PR TITLE
Publish updates from master

### PR DESCRIPTION
**Pull request by usha-mandya**
Publish updates from master

### CODE REVIEW

**Review by thaJeztah — APPROVED** [review]
LGTM

File: cloud/aci-integration.md [hunk 1]
```diff
@@ -99,16 +99,16 @@ There are two ways to use your new ACI context. You can use the `--context` flag
 docker --context myacicontext run -p 80:80 nginx
 ```
 
-Or, you can change context using `docker context use` to select the ACI context to be your focus for running Docker commands. For example, we can use the `docker context use` command to deploy an ngnix container:
+Or, you can change context using `docker context use` to select the ACI context to be your focus for running Docker commands. For example, we can use the `docker context use` command to deploy an Nginx container:
 
 ```console
 docker context use myacicontext
-docker run -p 80:80 nginx
+docker run -p 80:80 Nginx
```
Comment (StefanScherer) [inline cloud/aci-integration.md:106]: This leads to an error.
```
docker: invalid reference format: repository name must be lowercase.
```

**Review by StefanScherer — CHANGES_REQUESTED** [review]
Found one issue with `Nginx` that should be fixed first.

File: cloud/aci-integration.md (hunk 1 above)
Comment (usha-mandya) [reply cloud/aci-integration.md:106]: Good catch . Let me fix this.

Comment (usha-mandya) [reply cloud/aci-integration.md:106]: @StefanScherer PTAL - https://github.com/docker/docker.github.io/pull/12823

Comment (thaJeztah) [reply cloud/aci-integration.md:106]: oh, nice catch 👍

**Review by thaJeztah — DISMISSED** [review]
LGTM

**Review by StefanScherer — DISMISSED** [review]
LGTM

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**